src/App.js

```diff
@@ -1,24 +1,26 @@
 import React from "react";
-import Players from "./components/Players";
-import User from "./components/User";
```

Comment:
Please remove in p.json
    "@aws-sdk/client-dynamodb": "^3.150.0",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.3.0",
    "@testing-library/user-event": "^13.5.0",
    "aws-sdk": "^2.1199.0",
    "eslint": "^8.21.0"
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
Reasearch if need remove?
      "web-vitals": "^2.1.4"
  "@emotion/react": "^11.10.0",
    "@emotion/styled": "^11.10.0",

Move to deps:
    "react-router-dom": "^6.3.0"